lib/created_message_queue_handler.ts

```diff
@@ -167,6 +183,49 @@ function getEmailAddressFromProfile(
   }));
 }
 
+/**
+ * Try to create (save) a new notification
+ */
+async function createNotification(
+  lNotificationModel: NotificationModel,
+  senderMetadata: CreatedMessageEventSenderMetadata,
+  newMessageWithContent: NewMessageWithContent,
+  channel: NotificationChannel,
+  notificationChannel: NotificationChannelMeta
+): Promise<Either<RuntimeError, Option<NotificationEvent>>> {
+  const newNotification = createNewNotification(
+    ulidGenerator,
+    newMessageWithContent.fiscalCode,
+    newMessageWithContent.id
+  );
+
+  const errorOrNotification = await lNotificationModel.create(
+    {
+      ...newNotification,
+      channels: {
```

Comment:
we is this attribute plural if it always holds one single channel?

Reply:
renamed